ReplayKitExample/ReplayKitExample/ReplayKitVideoSource.swift

```diff
@@ -219,6 +253,85 @@ class ReplayKitVideoSource: NSObject, TVIVideoCapturer {
         }
         to.consumeCapturedFrame(frame)
         lastTimestamp = timestamp
+
+        // Frame retransmission logic.
+        if (ReplayKitVideoSource.retransmitLastFrame) {
+            if ReplayKitVideoSource.useDispatchQueue {
+                lastFrameStorage = frame
+                lastTransmitTimestamp = CMClockGetTime(CMClockGetHostTimeClock())
+                dispatchRetransmissions()
+            } else {
+                lastFrame = frame
+                if retransmitTimer == nil {
+                    scheduleRetransmissions()
+                }
+            }
+        }
+    }
+
+    func scheduleRetransmissions() {
```

Comment:
This approach has some problems.

1. Absolute timestamps don't match what comes out of ReplayKit, you might get new frames from the past (earlier than a retransmit). `dispatchRetransmissions` should be better at this.
2. There is no rescheduling of the timer on a per-frame basis, making the first retransmission a bit jagged.